runtime/laos/src/currency.rs

```diff
@@ -3,25 +3,19 @@ use laos_primitives::Balance;
 // Unit = the base number of indivisible units for balances
 // 18 decimals
 pub(crate) const UNIT: Balance = 1_000_000_000_000_000_000;
-pub(crate) const MILLIUNIT: Balance = UNIT / 1000;
+pub(crate) const MILLIUNIT: Balance = 1_000_000_000_000_000;
```

Comment:
I think previous version is more intuitive and clearer. `MILLI` is one-thousandth of a UNIT, so clearly dividing by 1000 is more readable